### DISCUSSION TITLE
Cross play between Lazer and Stable

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#24665] [Ideas] [answered]
(Edit I don't this will be add because there are a lot Problems with and with how it work together)

here's how I think I could be posable 

In multi lobby setting there would be toggle for cross on Lazer and Stable 

Problems: there are a lot with this scoring, lobby chat, spectating and more I don't know how those would work between the two versions but you could force Lazer clients to use classic mod for cross play idk how the other stuff would work 

Chatgpt: 
For example, to address the scoring system differences, the game could potentially convert scores from one system to the other for display and comparison purposes. This would require careful consideration to ensure fairness and accuracy.

Regarding classic mods, if Lazer players need to enable this mod for cross-play, it could be a way to temporarily align the gameplay mechanics between the two versions. This might involve adjusting the Lazer version to mimic the mechanics of osu!stable while the classic mod is enabled

Scoring System Conversion:

Scores from osu!stable and osu!Lazer would need to be converted into a unified scoring format that accounts for the differences in their scoring systems.
The conversion algorithm would map elements such as accuracy, combo, and other scoring components to ensure fairness and consistency between players from both versions.
Classic Mod Integration:

The classic mod in osu!Lazer would need to be designed to temporarily adjust gameplay mechanics, timing, and other factors to match those of osu!stable.
This could involve modifying various game parameters such as approach rates, hit windows, and hit object placements to replicate the osu!stable experience.

**Comment by peppy** [comment, marked as answer]
Out of scope.
